src/database.py

```diff
@@ -44,7 +45,64 @@
 )
 from utils import filter_forecast_values, floor_30_minutes_dt, get_start_datetime
 
-db_conn = DatabaseConnection(url=os.getenv("DB_URL", "not_set"))
```

Comment:
we probably cant merge this, as we wont have the real data base connection. Was this just to try something and see if it works?